### PR TITLE
NAS-130848 / 25.04 / Add apps stats

### PR DESCRIPTION
**Pull request by denysbutenko**
**Changes:**

Add apps stats on installed apps 

**Testing:**

Check installed apps page

### Downstream

|Affects         |Reasoning
|----------------|-------------------------------
|Documentation   | Adds back app stats.

### CODE REVIEW

**Review by undsoft — APPROVED** [review]
Looks good overall.

**Review by AlexKarpov98 — APPROVED** [review]

File: src/app/pages/dashboard/widgets/apps/common/app-network-info/app-network-info.component.html [hunk 1]
```diff
@@ -4,13 +4,13 @@ <h4>{{ 'Network I/O' | translate }}</h4>
     <div class="in-out-row">
       <span>{{ 'In' | translate }}:</span>
       <span *ixWithLoadingState="stats() as stats">
-        {{ stats.network.incoming | ixNetworkSpeed }}
+        {{ stats.networks[0].rx_bytes | ixNetworkSpeed }}
```
[the remark this answers is not in the document]
Comment (denysbutenko) [reply src/app/pages/dashboard/widgets/apps/common/app-network-info/app-network-info.component.html:7]: I noted this in the [ticket](https://ixsystems.atlassian.net/browse/NAS-131073)

File: src/app/pages/apps/components/installed-apps/app-row/app-row.component.html [hunk 2]
```diff
@@ -23,6 +23,31 @@
...
+  @if (stats()?.memory) {
+    <span>{{ stats().memory | ixFileSize }}</span>
+  } @else {
+    {{ 'N/A' | translate }}
+  }
+</div>
+<div class="cell cell-network" [matTooltip]="'Incoming / Outgoing network traffic' | translate">
+  @if (stats()?.networks) {
+    <span>
+      {{ stats().networks[0].rx_bytes | ixNetworkSpeed }}
```
Comment (undsoft) [inline src/app/pages/apps/components/installed-apps/app-row/app-row.component.html:43]: Commented in incorrect place. App widgets can be fixed later, but network stats need to be fixed here now.

**Review by undsoft — CHANGES_REQUESTED** [review]

**Review by undsoft — APPROVED** [review]
Looks good, thanks.

**Pull request merged**